diagnostics/TC_Rain/TC_rain.py

```diff
@@ -0,0 +1,299 @@
+# This file is part of the TC Rain Diagnostic POD of the MDTF code package (see mdtf/MDTF-diagnostics/LICENSE.txt)
```

Comment:
Could you please apply the Black code formatter to your .py scripts?  I appreciate this is not in the guide, but it is very helpful!

https://black.readthedocs.io/en/stable/